pkg/detectors/falsepositives_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build detectors
```

Comment:
As far as I can tell, nothing in this test relies on 'real' values. It would be beneficial for this test to be included in `make test` and `make test-community`.

Reply:
@zricethezav are you aware of any non-obvious reason that this test is behind `detectors`? (Also tagging @abmussani for visibility)

Reply:
> @zricethezav are you aware of any non-obvious reason that this test is behind detectors? (Also tagging @abmussani for visibility)

negative, seems like this test should be part of the community PR tests